styles.css

```diff
@@ -0,0 +1,646 @@
+* {
+    margin: 0;
+}
+
+body {
+    display: grid;
+}
+
+
+.layout {
+    background-color: white;
+    display: grid;
+    height: 105vh;
+    grid-template: 
+        "left-col        main-col        right-col"     1fr/
```

Comment:
no need to space things out so much try to keep it reasonably spaced e.g. "left-col  main-col  right-col" etc